libraries/Zigbee/src/ZigbeeEP.cpp

```diff
@@ -157,22 +175,32 @@ char *ZigbeeEP::readModel(uint8_t endpoint, uint16_t short_addr) {
   // clear read model
   _read_model = nullptr;
 
+  esp_zb_lock_acquire(portMAX_DELAY);
   esp_zb_zcl_read_attr_cmd_req(&read_req);
+  esp_zb_lock_release();
 
   //Wait for response or timeout
-  //Semaphore take
-  if (xSemaphoreTake(lock, portMAX_DELAY) != pdTRUE) {
+  if (xSemaphoreTake(lock, ZB_CMD_TIMEOUT) != pdTRUE) {
     log_e("Error while reading model");
   }
   return _read_model;
 }
 
-void ZigbeeEP::printBoundDevices() {
-  log_i("Bound devices:");
-  for ([[maybe_unused]]
-       const auto &device : _bound_devices) {
-    log_i("Device on endpoint %d, short address: 0x%x", device->endpoint, device->short_addr);
-    print_ieee_addr(device->ieee_addr);
+void ZigbeeEP::printBoundDevices(Print *print) {
```

Comment:
Such functions in Arduino should use `Print & print` instead.

Reply:
@me-no-dev But can I then use a default parameter? So if there is nothing passed, it will print in log_i?
`void printBoundDevices(Print *print = nullptr);`

Reply:
Done